lib/svgo/coa.js

```diff
@@ -112,6 +117,16 @@ async function action(args, opts, command) {
     }
   }
 
+  if (opts.eol != null) {
+    const eol = opts.eol.toLowerCase();
```

Comment:
Remove this too please. Better have configuration consistent.

Reply:
Whoops, forgot about this file! Addressed in https://github.com/svg/svgo/pull/1546/commits/592c01fdf17ca38c87ce0b02fabb62dbd206225e.